pallets/schemas/README.md

```diff
@@ -0,0 +1,79 @@
+| ----------------------------------------------------------------------------------------------------- | ----------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `set_max_schema_model_bytes`<br />Governance action to alter the maximum byte length of Schema models | Governance                                      | Tokens  | [`SchemaMaxSizeChanged`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaMaxSizeChanged) | 1             |
+| `propose_to_create_schema_v2`<br />Creates a Council proposal a new schema to the Frequency Council   | Token Account                                   | Tokens  | [`Proposed`](https://paritytech.github.io/polkadot-sdk/master/pallet_collective/pallet/enum.Event.html#variant.Proposed)                 | 66            |
+| `create_schema_via_governance_v2`<br />Governance called version of `create_schema_v3`                | Frequency Council                               | Tokens  | [`SchemaCreated`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaCreated)               | 66            |
+| `create_schema_v3`<br />Creates a new Schema.                                                         | Mainnet: Governance<br />Testnet: Token Account | Tokens  | [`SchemaCreated`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaCreated)               | 1             |
+| `propose_to_create_schema_name`<br />Creates a Council proposal to set the name of a Schema           | Token Account                                   | Tokens  | [`Proposed`](https://paritytech.github.io/polkadot-sdk/master/pallet_collective/pallet/enum.Event.html#variant.Proposed)                 | 1             |
+| `create_schema_name_via_governance`<br />Governance called to set the name of a Schema                | Frequency Council                               | Tokens  | [`SchemaNameCreated`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaNameCreated)       | 66            |
+
+See [Rust Docs](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/struct.Pallet.html) for more details.
+
+### State Queries
+
+| Name                              | Description                                                         | Query                            | Runtime Added |
+| --------------------------------- | ------------------------------------------------------------------- | -------------------------------- | ------------- |
+| Get Max Current Schema Identifier | Fetch current Schema Identifier maximum                             | `currentSchemaIdentifierMaximum` | 1             |
```

Comment:
should these also be linked to the Rustdocs like the other two tables?

Reply:
1. The links take up sooooo much room that the table becomes unreadable in markdown.
2. The rustdocs for these are really unhelpful because of the generated code. I didn't see any value outside of the link at the bottom: https://frequency-chain.github.io/frequency/pallet_msa/pallet/storage_types/index.html